### PR TITLE
Items crud

### PR DESCRIPTION
**Pull request by reid-andrew**
User stories 17 & 18
- Items index page - photos are clickable links to item show pages
- Item index page - inactive items are now hidden
- Item index page - Displays top/bottom 5 items based on qty purchased.

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -5,7 +5,10 @@ def index
       @merchant = Merchant.find(params[:merchant_id])
       @items = @merchant.items
     else
-      @items = Item.all
+      @items = Item.active_items
```
Comment (mlynch5187) [inline app/controllers/items_controller.rb:8]: Great job getting story 18 taken care of here.

File: app/models/item.rb [hunk 2]
```diff
@@ -12,6 +12,29 @@ class Item <ApplicationRecord
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
 
+  def self.active_items
+    Item.where(active?:true)
+  end
+
+  def self.top_five_items
```
Comment (mlynch5187) [inline app/models/item.rb:19]: Again, good puzzle-solving.

**Review by mlynch5187 — APPROVED** [review]
Great job here, lots of good logic, and was fairly RESTful and DRY.

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -5,7 +5,10 @@ def index
...
+      @top_five_items = Item.top_five_items
+      @bottom_five_items = Item.bottom_five_items
+
     end
```
Comment (javier-aguilar) [inline app/controllers/items_controller.rb:12]: When we refactor later on, we should try to send fewer objects to the view.

**Review by javier-aguilar — APPROVED** [review]
Make sure to do model testing for the Item instance methods, I didn't see any.  Overall, everything looks solid

**Pull request merged**